### ISSUE TITLE
animateOnDataChange causes chart not to render

### ISSUE THREAD

**Issue by KraljSamo** [Abhinandan-Kushwaha/react-native-gifted-charts#608]
When using animateOnDataChange it causes the chart lines not to render at all. The axes are visible. 

Minimal code example:

```
      <View style={{ backgroundColor: "white" }}>
        <LineChart
          isAnimated
          data={data}
          // animateOnDataChange
          animationDuration={1000}
          onDataChangeAnimationDuration={300}
        />
      </View>
```

I tried this on both POCO F3 Android phone and on iPhone 13, on both it happens the same. 
I tried this on 1.4.7 and 1.4.10 versions of this library. 

```
    "react-native": "0.73.6",
     "react-native-gesture-handler": "2.16.0",
     "react-native-gifted-charts": "1.4.7",
     "react-native-linear-gradient": "2.8.3",
     "react-native-reanimated": "3.8.1",
     "react-native-svg": "15.1.0",
    "react-native-svg-icon": "0.10.0",
```

**Comment by piotr-weareconflict** [comment]
Same issue, any solutions?

**Comment by Abhinandan-Kushwaha** [comment]
Hi @KraljSamo @piotr-weareconflict 
This has been fixed from versions `1.4.7` onwards 🎉
Please use the latest version of the library.
